.azure-pipelines.yml

```diff
@@ -269,6 +269,34 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt install python python-pip
+      pip install twine
+    displayName: 'Bootstrap'
+  - bash: |
+      set -eux
+      python setup.py generate_pyqt sdist
```

Comment:
`generate_pyqt` shouldn't be needed here (calling `python setup.py bdist_wheel` should depend on it automatically)

However I didn't see such dependency in the code, so if it is not in place I think it's better to have this on multiple lines:
```python
python setup.py sdist
python setup.py generate_pyqt  # Needed before bdist_wheel
python setup.py bdist_wheel
```
Here I considered sdist doesn't need this cooking (I'm not sure it is the case...)

We may want to have a deeper look at the setup.py configuration in a further PR ;-)

Reply:
Yup, I wanted to keep that one short, and didn't took time to correct the dependencies in setup.py , but it's true we should absolutely clean that one

I still prefer to generate the widgets before the sdist, and doing everything in one line seems to be ok

Reply:
Actually generate_pyqt is useless, widgets are generated nevertheless